packages/client/yggdrasil/yggdrasil.spec

```diff
@@ -0,0 +1,97 @@
+%define debug_package %{nil}
+
+%global go_arches x86_64 s390x ppc64le
+
+Name:    yggdrasil
+Version: 0.2.0
+Release: 3%{?dist}
+Epoch:   1
```

Comment:
this is so that our package always wins against which other package?

Reply:
I don't see an `yggdrasil` package in CentOS or EPEL, so not sure why this is set.

Reply:
https://github.com/theforeman/foreman-packaging/pull/7172#issuecomment-950464199

Reply:
No, this explains why you had to add the epoch to the changelog entry, but not why the epoch is used to begin with.

Reply:
Ah, I misread the question. No idea to be honest, it was already there when I pulled the rpm spec from rhc

Reply:
Hmm. So I *think* this is because there existed a `rhc` in the past that was totally unrelated to this new package, and downstream wanted to start versioning "fresh".

This doesn't apply to the `yggdrasil` name we're using here.

So I'd be in favor of dropping the Epoch (I *hate* Epochs), but also wouldn't block on this.

@ekohl @zjhuntin @Odilhao -- what do y'all think?

Reply:
If there's no need for an epoch, I'd avoid it. Epoch should only be used as a workaround.

Reply:
Dropped it